pages/project-version/pdf/index.jsx

```diff
@@ -32,7 +34,7 @@ module.exports = settings => {
           headerTemplate: header,
           footerTemplate: footer,
           margin: {
-            top: 100,
+            top: hasStatusBanner() ? 180 : 100,
```

Comment:
I can get on board with saving the result of the condition to a single-use variable, but having a function which always returns the same value seems particularly redundant

Reply:
Yeah had a bit of a brain fart there, not sure why I used a function.